index.js

```diff
@@ -20,6 +20,10 @@ module.exports = {
   extends: ['skyscanner', 'prettier', 'prettier/react'],
   plugins: ['prettier'],
   rules: {
+    'import/order': ['error', {
+      'groups': ['builtin', 'external', 'parent', 'sibling', 'index'],
```

Comment:
The groups here are just an initial proposal, this is the default order in eslint: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#groups-array

The order in airbn is a bit different, although they have a `TODO` there to make it more strict:
https://github.com/airbnb/javascript/blob/d951220399c75099838fc79577033b00530b2b90/packages/eslint-config-airbnb-base/rules/imports.js#L146